src/pkg/reg/adapter/awsecr/adapter.go

```diff
@@ -64,10 +64,10 @@ func newAdapter(registry *model.Registry) (*adapter, error) {
 
 func parseAccountRegion(url string) (string, string, error) {
 	rs := ecrRegexp.FindStringSubmatch(url)
-	if rs == nil {
+	if rs == nil || len(rs) < 3 {
```

Comment:
Since the following logic extract the `rs[3]`, so it may be necessary to use `< 4` here instead?